packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -58,15 +77,45 @@ class MenuBase extends React.Component<MenuProps, MenuState> {
       ouiaId,
       ouiaSafe,
       containsFlyout,
+      containsDrilldown,
+      isMenuDrilledIn,
+      drilldownItemPath,
+      drilledInMenus,
+      onDrillIn,
+      onDrillOut,
+      onGetMenuHeight,
+      parentMenu = null,
       activeItemId = null,
       innerRef,
       ...props
     } = this.props;
-
+    const _isMenuDrilledIn =
+      (isMenuDrilledIn && isMenuDrilledIn) || (drilledInMenus && drilledInMenus.includes(id)) || false;
```

Comment:
This looks a bit funny, should it be `isMenuDrilledIn !== undefined && isMenuDrilledIn || ...`

Reply:
That should be equivalent